cli/tests/pcluster/config/test_validators.py

```diff
@@ -719,6 +794,92 @@ def test_fsx_validator(mocker, boto3_stubber, section_dict, bucket, expected_err
     utils.assert_param_validator(mocker, config_parser_dict, expected_error=expected_error)
 
 
+@pytest.mark.parametrize(
+    "section_dict, expected_error",
+    [
+        ({"fsx_fs_id": "fs-0123456789abcdef0", "shared_dir": "/fsx"}, None),
+        (
+            {"fsx_fs_id": "fs-0123456789abcdef0", "shared_dir": "/fsx", "storage_capacity": 3600},
+            "storage_capacity is ignored when specifying an existing Lustre file system",
+        ),
+    ],
+)
+def test_fsx_ignored_parameters_validator(mocker, boto3_stubber, section_dict, expected_error):
```

Comment:
Isn't this test too complex to just check the logic of the fsx_ignored_parameters_validator?
Maybe this is due to the fact that all fsx validators are still involved here, since it uses the ``utils.assert_param_validator``.
I'd either check directly the result of a specific call to ``fsx_ignored_parameters_validator`` or move these checks into the ``test_fsx_validator`` method. Does it make sense?

Reply:
I completely agree. It's because of the complexity contained in `fsx_id_validator`. Any test that specifies that parameter will need to do this mocking. It would be possible to handle this in that same validator, but it's already so complex as it is.

As for calling it directly, I looked into this and the mocking for this gets ugly as well, so in my opinion we're just shifting complexity rather than eliminating it.

For those reasons I'd like to leave this as is.